src/aat/resources/features/F-052/S-070.td.json

```diff
@@ -0,0 +1,18 @@
+{
+	"_guid_": "S-070",
+	"_extends_": "F-052_Test_Data_Base",
+
+	"title": "must return 200 and list of case data for the given search criteria",
+
+	"specs": [
+		"an active profile in CCD",
+		"contains correct input path variables",
+		"contains the case details, along with a HTTP 200 OK"
+	],
+	"expectedResponse": {
+		"_extends_": "Common_200_Response",
+    "body" : {
+      "arrayInMap" : "[[ANY]]"
```

Comment:
Considering that we may have multiple cases for this particular search criteria, accepting a generic response and relying on the response code.

Reply:
Yes, there is nothing more we can do because this is an Array and we can't control the number of items as well as their order in the array, so this is the only option available until this is handled by framework.